app/views/users/home.html.erb

```diff
@@ -33,23 +33,30 @@
 <%= form_with(url: '/users/home', method: 'get', local: true) do %>
   <div class="form-group">
     <label for="path">Path:</label>
-    <input id="path" type="text" name="path" placeholder="/rest/v1.0/me"></input>
+    <input id="path" type="text" name="path" placeholder="companies/id">
+  </div>
+
+  <div class="form-group">
+    <label for="version">Version:</label>
+    <input id="version" type="text" name="version" placeholder="v1.0">
   </div>
 
   <div class="form-group">
     <label for="body">Body:</label>
-    <input id="body" type="textarea" name="body" placeholder="{}"></input>
+    <input id="body" type="textarea" name="body" placeholder="{}">
```

Comment:
Did you mean to use `<textarea>...</textarea>`?

Reply:
That looks better. Apparently `type="textarea"` doesn't work.